posthog/api/test/__snapshots__/test_action.ambr

```diff
@@ -20,6 +20,7 @@
          "posthog_user"."anonymize_data",
          "posthog_user"."toolbar_mode",
          "posthog_user"."is_email_verified",
+         "posthog_user"."requested_password_reset_at",
```

Comment:
It seems like a waste of resources to load this anywhere outside of `autentication.py`, maybe we can make this deferred on `UserManager`, like `TeamManager` defers some deprecated fields.